tests/testChebyValues.py

```diff
@@ -87,28 +90,35 @@ def testGetEphemerides(self):
         chebyValues = ChebyValues()
         # chebyValues.setCoefficients(self.chebyFits)
         chebyValues.readCoefficients(self.coeffFile)
+        # Single time, halfway through interval.
         time = self.tStart + self.interval / 2.0
-        # Test for all objects.
+        # Test for a single time, but all the objects.
         ephemerides = chebyValues.getEphemerides(time)
         pyephemerides = self.pyephems.generateEphemerides(time, obscode=807,
                                                           timeScale='TAI', byObject=False)
         # RA and Dec should agree to 2.5mas (skyTolerance above)
-        pos_residuals = np.sqrt((ephemerides['ra'] - pyephemerides['ra'][0]) ** 2 +
-                                (ephemerides['dec'] - pyephemerides['dec'][0]) ** 2)
+        pos_residuals = np.sqrt((ephemerides['ra'][:,0] - pyephemerides['ra'][0]) ** 2 +
+                                ((ephemerides['dec'][:,0] - pyephemerides['dec']) *
+                                 np.cos(np.radians(ephemerides['dec'][:,0]))) ** 2)
         pos_residuals *= 3600.0 * 1000.0
-        self.assertTrue(np.max(pos_residuals) <= 2.5)
         # Let's just look at the max residuals in all quantities.
         for k in ('ra', 'dec', 'dradt', 'ddecdt', 'delta'):
-            resids = np.abs(ephemerides[k] - pyephemerides[k][0])
+            resids = np.abs(ephemerides[k][:,0] - pyephemerides[k][0])
+            if k != 'delta':
+                resids *= 3600.0 * 1000.0
             print('max diff', k, np.max(resids))
         resids = np.abs(ephemerides['elongation'] - pyephemerides['solarelon'][0])
         print('max diff elongation', np.max(resids))
         resids = np.abs(ephemerides['vmag'] - pyephemerides['magV'][0])
         print('max diff vmag', np.max(resids))
-        # Test this for a subset of the objects.
+        self.assertTrue(np.max(pos_residuals) <= 2.5)
```

Comment:
Change this to `self.assertLess(np.max(pos_residuals), 2.5)`.  It will produce a better error message in the case of failure.